<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortLineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnexionStatusLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CelsiusPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FahrenheitPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HygrometriePushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygrométrie</string>
    </property>
   </widget>
   <widget class="QLabel" name="CelsiusLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FahrenheitLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HygrometrieLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnexionButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CelsiusPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCelsiusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FahrenheitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFahrenheitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HygrometriePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHygrometrieButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnexionButtonClicked()</slot>
  <slot>onValiderButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onCelsiusButtonClicked()</slot>
  <slot>onFahrenheitButtonClicked()</slot>
  <slot>onHygrometrieButtonClicked()</slot>
  <slot>onCelsiusReadyRead()</slot>
 </slots>
</ui>
